<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>791</width>
      <height>471</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(182, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Gestion des parents:</string>
     </attribute>
     <widget class="QTabWidget" name="parents">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>791</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(0, 0, 0);
font: 15pt &quot;MV Boli&quot;;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Fiches  parents</string>
       </attribute>
       <widget class="QTableView" name="tabparent">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>481</width>
          <height>231</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
text-decoration: underline;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_13">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>300</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
background-color: rgb(216, 198, 255);</string>
        </property>
        <property name="text">
         <string>Actualiser</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_14">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>340</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
background-color: rgb(216, 198, 255);</string>
        </property>
        <property name="text">
         <string>Afficher</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>320</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
</string>
        </property>
        <property name="text">
         <string>TRIER NOM</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>260</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
background-color: rgb(216, 198, 255);</string>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="CIN2_2">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>260</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>320</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
background-color: rgb(216, 198, 255);</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_4">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>280</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
</string>
        </property>
        <property name="text">
         <string>TRIER CIN</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_CIN_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>113</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SimSun</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>CIN :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SimSun</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Nom Parent:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>60</y>
          <width>113</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Prenom Parent:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prenom_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>100</y>
          <width>113</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Nom eleve:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="nomeleve1_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>140</y>
          <width>113</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_email_3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>180</y>
          <width>113</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Numero:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_num_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>220</y>
          <width>113</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_17">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;

color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
background-color: rgb(216, 198, 255);</string>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>270</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
background-color: rgb(216, 198, 255);</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>404</x>
          <y>302</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
background-color: rgb(216, 198, 255);
</string>
        </property>
        <property name="text">
         <string>Imprimer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Gestion des reclamations :</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>771</width>
        <height>401</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(0, 0, 0);
font: 15pt &quot;MV Boli&quot;;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>10</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SimSun</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
background-color: rgb(216, 198, 255);</string>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Num Reclamation :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>50</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Nom parent : </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>20</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Nom eleve :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>Proffeseur :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string>objet :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="num">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>20</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="parent">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="prof">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="objet">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>511</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="tablereclamation_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>471</width>
          <height>171</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_18">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>240</y>
          <width>81</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
background-color: rgb(216, 198, 255);</string>
        </property>
        <property name="text">
         <string>Actualiser</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_19">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>190</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SimSun</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;

</string>
        </property>
        <property name="text">
         <string>Trier num</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>240</y>
          <width>81</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SimSun</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
background-color: rgb(216, 198, 255);</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_12">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>240</y>
          <width>81</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SimSun</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
background-color: rgb(216, 198, 255);</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_20">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>190</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SimSun</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
</string>
        </property>
        <property name="text">
         <string>Trier nom</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_19">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>310</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>SimSun</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
background-color: rgb(216, 198, 255);</string>
        </property>
        <property name="text">
         <string>Rechercher num </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="num_2">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>310</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>20</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>90</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(42, 145, 255);
font: 10pt &quot;SimSun&quot;;
background-color: rgb(216, 198, 255);</string>
        </property>
        <property name="text">
         <string>Envoyer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>140</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>imprimer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
